src/ssl.c

```diff
@@ -18058,6 +18117,13 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
         }
     }
 
+    void (*wolfSSL_get_locking_callback(void))(int, int, const char*, int)
```

Comment:
Please define a callback type with `typedef` and use that here instead. It took me too long to figure out that this was returning a function.

Reply:
Or just use `mutex_cb` as the callback type.

Reply:
Good idea, fixed.